README.md

```diff
@@ -1,73 +1,24 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
+# Hello-microverse
 
+> This project is about to setup a html and css hellp word
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+html and css
 
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
- [ ] Kindly remove this and add a Getting started section like how to clone the repo and who to see on the browser
- [ ] Prerequisite: e.g vscode editor, a web browser(chrome, etc).